faq.html

```diff
@@ -97,8 +97,15 @@
 </div>
 
 <script type="text/javascript">
-    $('#faq-language-selection').show();
-
-    var userLang = (navigator.language || navigator.userLanguage).split('-')[0];
-    switchLanguage(userLang);
+    $('#faq-language-selection').show()
+    const parameters = location.search
+        .substr(1)
+        .split("&")
+        .reduce((obj, str) => {
```

Comment:
While I like the arrow function syntax, it is not supported in enough browsers AFAIK.

Reply:
[![image](https://user-images.githubusercontent.com/29176678/30206803-8714b210-948d-11e7-9956-c411a315a54a.png)](http://caniuse.com/#search=arrow%20function)

It seems like it's supported pretty well by most modern browsers.

Reply:
Except of the mobile area, especially older stock browsers and Opera Mini.

I just tested our current version with Opera Mini and it doesn't work either. OK, let's not support Opera Mini. ;)